converter/dicom-cast/samples/templates/default-azuredeploy.json

```diff
@@ -94,6 +94,13 @@
             "metadata": {
                 "description": "The endpoint of the FHIR server."
             }
+        },
+        "IgnoreSyncOfInvalidTagValue": {
+            "defaultValue": false,
```

Comment:
Sorry Abdullah, should we make this true by default?

Reply:
Updated